sdk/js/src/utils/consts.ts

```diff
@@ -149,40 +150,40 @@ const MAINNET = {
     token_bridge: undefined,
     nft_bridge: undefined,
   },
-  injective: {
-    core: undefined,
+  moonbeam: {
+    core: "0xC8e2b0cD52Cf01b0Ce87d389Daa3d414d4cE29f3",
     token_bridge: undefined,
     nft_bridge: undefined,
   },
-  osmosis: {
+  neon: {
     core: undefined,
     token_bridge: undefined,
     nft_bridge: undefined,
   },
-  aptos: {
-    core: undefined,
-    token_bridge: undefined,
+  terra2: {
```

Comment:
any reason for reordering these keys?

Reply:
My guess is that it happened during a rebase.  When I rebase again, I'll take whatever I get from dev.v2